src/include/drbd/global_conf.rb

```diff
@@ -54,6 +60,14 @@ def global_conf_GetDialog
                   Builtins.tointeger(@diagref)
                 )
               ),
+              Left(
+                CheckBox(
+                  Id("UdevAlwaysUseVnr"),
+                  Opt(:notify),
+                  _("Udev Always Use VNR"),
```

Comment:
I do not know what "VNR" means, I'm not familiar with DRBD, maybe you should use the full term here. Something like "Always Use XXXXXX (VNR) for Udev".

Reply:
It is the short for volume number. It is possible to have a VNR section in configuration, although not a common case.
I will change it.